pandas/core/style.py

```diff
@@ -0,0 +1,769 @@
+"""
+Module for applying conditional formatting to
+DataFrames and Series.
+"""
+from functools import partial
+from contextlib import contextmanager
+from uuid import uuid1
+import copy
+from collections import defaultdict
+
+try:
+    from jinja2 import Template
+    has_jinja = True
+except ImportError:
+    has_jinja = False
```

Comment:
I think this is leftover from when I included it in `pandas.core.api`, so we couldn't raise. I'll get rid of it now that we're not importing it by default.